bootstrap/kubeadm/config/default/manager_arguments.yaml

```diff
@@ -0,0 +1,4 @@
+- op: replace
+  path: "/spec/template/spec/containers/0/args"
+  value:
+    - "--enable-leader-election"
```

Comment:
We shouldn't need this here, this should be the default in config/manager/manager.yaml

Reply:
oh yeah, nice, i'll pull these out